frontend/packages/dev-console/integration-tests/support/commands/hooks.ts

```diff
@@ -19,7 +20,14 @@ before(() => {
   const bridgePasswordPassword: string = Cypress.env('BRIDGE_HTPASSWD_PASSWORD') || 'test';
   cy.login(bridgePasswordIDP, bridgePasswordUsername, bridgePasswordPassword);
   cy.document().its('readyState').should('eq', 'complete');
+  cy.exec(
+    `oc patch console.operator.openshift.io/cluster --type='merge' -p '{"spec":{"customization":{"perspectives":[{"id":"dev","visibility":{"state":"Enabled"}}]}}}'`,
```

Comment:
Is this only a stop-gap until we are able to update the tests?

We should be running most tests with default configuration if possible. I'd only expect a handful of tests to check the dev perspective works when enabled and the other tests be updated to use the admin perspective nav items to get to the same pages.

Reply:
Yes, I was thinking of this pr as a stop-gap to unblock the pr but as admin tests are also being updated I will update the tests to point to the admin side of things wherever it can be used

Reply:
Thanks @sanketpathak. There are a lot of test changes happening for the PF6 upgrade as well, so we should coordinate to avoid conflicts. @jhadvig @rhamilto FYI